database-models/src/src/main/java/service/health/check/models/Address.java

```diff
@@ -28,9 +30,27 @@
 	@Column(name = "port", length = 256, nullable = false)
 	private String port;
 
+	@Column(name = "first_admin", length = 256, nullable = false)
+	private String firstAdmin;
+
+	@Column(name = "second_admin", length = 256, nullable = false)
+	private String secondAdmin;
+
+	@Column(name = "send_notification_after", nullable = false)
+	private Integer sendNotificationAfter;
+
+	@Column(name = "resend_notification_after", nullable = false)
+	private Integer resendNotificationAfter;
+
+	@Column(name = "last_healthy")
+	private Timestamp lastHealthy;
+
+	@Column(name = "notification_sent")
+	private Timestamp notificationSent;
+
 	@Type(type= "org.hibernate.type.BooleanType")
-	@Column(name = "healthy", nullable = false)
-	private Boolean healthy;
+	@Column(name = "second_notification_sent")
```

Comment:
nullable = false

Reply:
zmieniłem jak to działa i to już nie jest Boolean, a Timestamp